docs/resources/Channel.md

```diff
@@ -232,51 +232,43 @@ Represents a message sent in a channel within Discord.
+| id                      | snowflake                                                                                                                          | id of the message                                                                                                                                |
+| channel_id              | snowflake                                                                                                                          | id of the channel the message was sent in                                                                                                        |
+| author\*                | [user](#DOCS_RESOURCES_USER/user-object) object                                                                                    | the author of this message (not guaranteed to be a valid user, see below)                                                                        |
+| content                 | string                                                                                                                             | contents of the message                                                                                                                          |
+| timestamp               | ISO8601 timestamp                                                                                                                  | when this message was sent                                                                                                                       |
+| edited_timestamp        | ?ISO8601 timestamp                                                                                                                 | when this message was edited (or null if never)                                                                                                  |
+| tts                     | boolean                                                                                                                            | whether this was a TTS message                                                                                                                   |
+| mention_everyone        | boolean                                                                                                                            | whether this message mentions everyone                                                                                                           |
+| mentions                | array of [user](#DOCS_RESOURCES_USER/user-object) objects                                                                          | users specifically mentioned in the message                                                                                                      |
```

Comment:
is `mentions` specific to gateway? it's included in both tables

Reply:
Hmmm yeah that's tricky...in that case maybe leaving it where you added it is fine. We could also keep that in the description or add a footnote to the `Channel.md` table with that information (and maybe an `INFO` callout above the gateway table that also the difference).

I slightly lean towards the option I described so that there's consistency with other tables (and they'll already be referencing the `Channel.md` table for other fields), but I'm also fine keeping it the way you have now.